great_expectations/validator/validator.py

```diff
@@ -1657,42 +1584,135 @@ def head(
+        expectation_suite_name: str = None,
+    ):
+        """Instantiates `_expectation_suite` as empty by default or with a specified expectation `config`.
+        In addition, this always sets the `default_expectation_args` to:
+            `include_config`: False,
+            `catch_exceptions`: False,
+            `output_format`: 'BASIC'
+
+        By default, initializes data_asset_type to the name of the implementing class, but subclasses
+        that have interoperable semantics (e.g. Dataset) may override that parameter to clarify their
+        interoperability.
+
+        Args:
+            expectation_suite (json): \
+                A json-serializable expectation config. \
+                If None, creates default `_expectation_suite` with an empty list of expectations and \
+                key value `data_asset_name` as `data_asset_name`.
+
+            expectation_suite_name (string): \
+                The name to assign to the `expectation_suite.expectation_suite_name`
+
+        Returns:
+            None
+        """
+        # Checking type of expectation_suite.
+        # Check for expectation_suite_name is already done by ExpectationSuiteIdentifier
+        if expectation_suite and not isinstance(expectation_suite, ExpectationSuite):
+            raise TypeError(
+                "expectation_suite must be of type ExpectationSuite, not {}".format(
+                    type(expectation_suite)
+                )
+            )
+        if expectation_suite is not None:
+            if isinstance(expectation_suite, dict):
+                expectation_suite = expectationSuiteSchema.load(expectation_suite)
+            else:
+                expectation_suite = copy.deepcopy(expectation_suite)
+            self._expectation_suite = expectation_suite
+
+            if expectation_suite_name is not None:
+                if (
+                    self._expectation_suite.expectation_suite_name
+                    != expectation_suite_name
+                ):
+                    logger.warning(
+                        "Overriding existing expectation_suite_name {n1} with new name {n2}".format(
+                            n1=self._expectation_suite.expectation_suite_name,
+                            n2=expectation_suite_name,
+                        )
+                    )
+                self._expectation_suite.expectation_suite_name = expectation_suite_name
+
+        else:
+            if expectation_suite_name is None:
+                expectation_suite_name = "default"
+            self._expectation_suite = ExpectationSuite(
+                expectation_suite_name=expectation_suite_name
+            )
+
+        self._expectation_suite.execution_engine_type = type(
+            self.execution_engine
+        ).__name__
+
+    def _get_runtime_configuration(
+        self,
+        catch_exceptions: Optional[bool] = None,
+        result_format: Optional[Union[dict, str]] = None,
```

Comment:
Should we include `include_config` here as well? Do we need `meta` here too?

Reply:
@talagluck If we do any functionality increase, then we need to do it in a separate PR.  Specifically to `include_config`, I would say "NO" -- because it does NOT impact validation and how the validation results are returned.  This is why it is not there to begin with, I believe.  Good question though (I thought about it too).  Thanks.

Reply:
I think we may need `include_config`, and if we do not include `include_config` here, we should have a comment as to why, since `include_config` is the third of three `runtime_kwargs`. It may also be worth looking at whether `include_config` should be a top level `runtime_kwarg`, as it might make more sense as a component of the `result_format` dictionary.

Reply:
Comment added for discussion and architectural review.